Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM photon:2.0 as builder
-MAINTAINER Venil Noronha <veniln@vmware.com>
-
-RUN tdnf install -y openssl-1.0.2o-3.ph2.x86_64
-RUN tdnf install -y go-1.10.3-1.ph2.x86_64
+FROM golang:1.12.1 as builder
```

Comment:
We still need to use the `photon` container due to licensing reasons. Can you install Go v1.12.1 by downloading a tarball instead?

Reply:
Done